Source/Scene/Cesium3DTileFeatureTableResources.js

```diff
@@ -0,0 +1,87 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/DeveloperError',
+        '../Renderer/WebGLConstants'
+    ], function(
+        defaultValue,
+        defined,
+        DeveloperError,
+        WebGLConstants) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function Cesium3DTileFeatureTableResources(featureTableJSON, featureTableBinary) {
+        this.json = featureTableJSON;
+        this.binaryDataView = new DataView(featureTableBinary);
+        this._cachedProperties = {};
+        this.featuresLength = 0;
+    }
+
+    Cesium3DTileFeatureTableResources.prototype.getGlobalProperty = function(semantic, componentType, count) {
```

Comment:
Not sure that "global" is the best word here; it's not really about scope.  It just means apply to all instances, not individual instances.  I don't have anything better right now though.  Will let you know if I think of something.